exercises/practice/word-count/word_count.h

```diff
@@ -26,4 +26,7 @@ typedef struct word_count_word {
 //           words will contain the results up to that point.
 int count_words(const char *sentence, word_count_word_t *words);
 
+// used to sort the words before comparison
+int cmp_word_count_word(const void *a, const void *b);
+
```

Comment:
I am not sure if this definition should live here, or if I create a new header file for the test.

Reply:
It would be best to leave it as a `static` function (basically a private function) in the test file. If you mark the definition private in `test_word_count.c` then you can remove this.

Reply:
```suggestion
```